templates/analytics.html

```diff
@@ -0,0 +1,41 @@
+{% extends "base.html" %}
+
+{% block title %}Analytics{% endblock %}
+
+{% block content %}
+  <link rel="stylesheet" type="text/css" href="{{url_for('static', filename='styles/style.css')}}" >
+  <link rel="stylesheet" type="text/css" href="{{url_for('static', filename='styles/base.css')}}" >
```

Comment:
sidebar and base ? and style? to many 'base' like csses make some cleaning keep only whats required in your pages